lib/teleterm/clusters/cluster_gateways_test.go

```diff
@@ -0,0 +1,45 @@
+package clusters
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gravitational/teleport/lib/client"
+	"github.com/gravitational/teleport/lib/defaults"
+	"github.com/gravitational/teleport/lib/teleterm/api/uri"
+	"github.com/gravitational/teleport/lib/teleterm/gateway"
+
+	"github.com/stretchr/testify/require"
+)
+
+type mockCLICommandProvider struct{}
+
+func (m mockCLICommandProvider) GetCommand(cluster *Cluster, gateway *gateway.Gateway) (*string, error) {
+	command := fmt.Sprintf("%s/%s", gateway.TargetName, gateway.TargetSubresourceName)
+	return &command, nil
+}
+
+func TestSetGatewayTargetSubresourceName(t *testing.T) {
```

Comment:
Added a small test so that at least there's something in there, more tests for lib/teleterm to come after we tackle #13278.